packages/analytics/lib/modular/index.js

```diff
@@ -569,6 +619,7 @@ export function isSupported() {
   // always return "true" for now until Web implementation. Web only.
   return Promise.resolve(true);
 }
+
```

Comment:
There are unnecessary spacing changes all over this diff, somewhere between 100 and 200 lines show up as visible changes in the PR diff viewer without representing any functional changes. This makes diffs harder to read and should be avoided